docs/user/metacat/source/submitting.rst

```diff
@@ -1,1006 +1,34 @@
-    <input type="hidden" name="action" value="login"> <input type=
-    "hidden" name="username" value=""> <input type="hidden" name=
-    "qformat" value="xml"> <input type="hidden" name=
-    "enableediting" value="false">
-
-    <table>
-      <tr valign="middle">
-        <td align="left" valign="middle" class="text_plain">
-        username:</td>
-
-        <td width="173" align="left" class="text_plain" style=
-        "padding-top: 2px; padding-bottom: 2px;"><input name="uid"
-        type="text" style="width: 140px;" value=""></td>
-      </tr>
 
-      <tr valign="middle">
-        <td height="28" align="left" valign="middle" class=
-        "text_plain">organization:</td>
+MetacatUI is the default web-interface that is shipped with Metacat.
 
-        <td align="left" class="text_plain" style=
-        "padding-top: 2px; padding-bottom: 2px;"><select name=
-        "organization" style="width:140px;">
-          <option value=""    selected>&#8212; choose one &#8212;</option>
-          <option value="NCEAS">NCEAS</option>
-          <option value="LTER">LTER</option>
-          <option value="UCNRS">UCNRS</option>
-          <option value="PISCO">PISCO</option>
-          <option value="OBFS">OBFS</option>
-          <option value="OSUBS">OSUBS</option>
-          <option value="SAEON">SAEON</option>
-          <option value="SANParks">SANParks</option>
-          <option value="SDSC">SDSC</option>
-          <option value="KU">KU</option>
-          <option value="unaffiliated">unaffiliated</option>
-        </select></td>
-      </tr>
+To begin submitting data, navigate to `metacatui` in your internet browser at:
 
-      <tr valign="middle">
-        <td width="85" align="left" valign="middle" class=
-        "text_plain">password:</td>
+    ``https://<yourhost.org>/metacatui``
 
-        <td colspan="2" align="left" class="text_plain" style=
-        "padding-top: 2px; padding-bottom: 2px;">
-          <table width="100%" border="0" cellpadding="0"
-          cellspacing="0">
-            <tr>
-              <td width="150" align="left"><input name="password"
-              type="password" maxlength="50" style="width:140px;"
-              value=""></td>
+Then click 'Sign in' in the upper right hand corner. A modal will appear with a button to sign in.
+In order to sign in, you will need an ``ORCID iD``. If you do not have one, you can sign up for
+free at https://orcid.org - after which you will receive a 16-digit ORCID iD.
 
-              <td align="center" class="buttonBG_login">
-              <input type="submit" name="loginAction" value="Login"
-              class="button_login"></td>
+Once you've registered and/or signed in, you will be returned to ``metacatui``. Move your cursor to
+the upper right-hand corner and hover above your name, which will reveal a drop down menu. Click
+'Submit Data', and add your data and metadata documents & fill in the relevant and required fields.
```

Comment:
a picture may be worth 1,000 words here :-)

Reply:
I have added a screenshot